pkg/executables/config/kind.yaml

```diff
@@ -19,6 +19,25 @@ kubeadmConfigPatches:
         imageTag: {{.EtcdVersion}}
     imageRepository: {{.KubernetesRepository}}
     kubernetesVersion: {{.KubernetesVersion}}
+{{- if (ne .RegistryMirrorEndpoint "") }}
+containerdConfigPatches:
+  - |
+    [plugins."io.containerd.grpc.v1.cri".registry.mirrors]
```

Comment:
Is it necessary to allow users to pass authentication config as well?

```
[plugins."io.containerd.grpc.v1.cri".registry.configs."{{.RegistryMirrorEndpoint}}".auth]
username = ""
  password = ""
  auth = ""
  identitytoken = ""
```

Reply:
I think it is ok to not add this. In on prem environments it would be a valid requirement to say that core eksa images must reside in a non authenticated repo/project